src/Build/BackEnd/Components/SdkResolution/SdkResolverManifest.cs

```diff
@@ -47,22 +62,34 @@ internal static SdkResolverManifest Load(string filePath)
             return null;
         }
 
+        // This parsing code is very specific and not forward compatible, but it should be all right.
```

Comment:
```suggestion
        // This parsing code is very specific and not forward compatible, but since resolvers generally ship in the same release vehicle as MSBuild itself, only backward compatibility is required.
```